crates/bevy_audio/src/audio_output.rs

```diff
@@ -41,25 +42,39 @@ impl<Source> AudioOutput<Source>
 where
     Source: Asset + Decodable,
 {
-    fn play_source(&self, audio_source: &Source) {
+    fn play_source(&self, audio_source: &Source, repeat: bool) -> Option<Sink> {
```

Comment:
I'm not sure how I feel about using a bool here. Aren't there more options that we might want to use in the future? Are we going to simply keep adding flags if that's the case?